3-platform/order/spec/src/main/java/com/_6core/platform/orderspec/rest/v1/dto/OrderResponse.java

```diff
@@ -0,0 +1,16 @@
+package com._6core.platform.orderspec.rest.v1.dto;
+
+import com._6core.lib.java.domain.model.order.OrderItemV01;
+import com._6core.lib.java.domain.model.order.OrderV02;
+import java.math.BigInteger;
+import java.util.Set;
+
+public record OrderResponse(
+        String orderId, 
+        String status, 
+        BigInteger total, 
+        Set<OrderItemV01> orderItems, 
+        String userId)
+        implements OrderV02 {
```

Comment:
OrderV02 is not implemented yet
```suggestion
        implements OrderV01 {
```